### PR TITLE
Release v1

### PR DESCRIPTION
**Pull request by Thomasdezeeuw**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+* Implement `AsFd` for`TcpListener`, `TcpStream`, `UdpSocket`, `UnixDatagram`,
+  `UnixListener`, `UnixStream`, `pipe::Receiver` and `pipe::Sender`
+  (https://github.com/tokio-rs/mio/pull/1749, https://github.com/tokio-rs/mio/pull/1797).
+* Implement `From` for `TcpListener`, `TcpStream`, `UdpSocket`, `UnixDatagram`,
+  `UnixListener`, and `UnixStream` for their standard library counterpart
+  (https://github.com/tokio-rs/mio/pull/1767).
+* Add support for abstract namespaces on Android
+  (https://github.com/tokio-rs/mio/pull/1749).
+* Add support for QNX OS
+  (https://github.com/tokio-rs/mio/pull/1766).
```
Comment (Thomasdezeeuw) [inline CHANGELOG.md:18]: Once merged.
```suggestion
  (https://github.com/tokio-rs/mio/pull/1766, https://github.com/tokio-rs/mio/pull/1800).
```

**Comment by Thomasdezeeuw** [comment]
Note to self: this is update to with https://github.com/tokio-rs/mio/commit/5200f6cbbfeaf45843af334652ccfd3ec7c7bf06.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,43 @@
+# 1.0
+
+With v1 Mio is able to bump it's MSRV, allowing us to implement I/O safety
```
Comment (atouchet) [inline CHANGELOG.md:3]: That apostrophe should be removed.

**Review by Darksonn — APPROVED** [review]

**Comment by Thomasdezeeuw** [comment]
Release to crates.io: https://crates.io/crates/mio/1.0.0.

**Pull request merged**